xrpl4j-core/src/test/resources/codec-fixtures.json

```diff
@@ -4013,7 +4061,9 @@
         "LedgerEntryType": "DirectoryNode",
         "Flags": 0,
         "RootIndex": "E2EC9E1BC7B4667B7A5F2F68857F6E6A478A09B5BB4F99E09F694437C4152DED",
-        "Indexes": ["65492B9F30F1CBEA168509128EB8619BAE02A7A7A4725FF3F8DAA70FA707A26E"]
+        "Indexes": [
```

Comment:
Agreed, this is what IntelliJ did when I reformatted the file. I would vote for keeping this new formatting so that we can have IntelliJ reformat the file in the future without creating a bunch of changes, but am happy to revert the formatting changes if it makes things easier to review.

Reply:
Agreed.